aiogram_dialog/manager/registry.py

```diff
@@ -18,49 +17,27 @@
 
 
 class DialogEventObserver(TelegramEventObserver):
-    def _resolve_middlewares(self, *, outer: bool = False) -> List[MiddlewareType]:
-        """
-        Get all middlewares in a tree
-        :param *:
-        """
-        middlewares = []
-        if outer:
-            middlewares.extend(self.outer_middlewares)
-        else:
-            for router in reversed(list(self.router.chain_head)):
-                if router.observers.get(self.event_name):
-                    observer = router.observers[self.event_name]
-                    middlewares.extend(observer.middlewares)
-
-        return middlewares
-
-
-class DialogRouter(Router):
-    def __init__(self, **kwargs: Any):
-        super(DialogRouter, self).__init__(**kwargs)
-        self.aiogd_update = self.observers[DIALOG_EVENT_NAME] = DialogEventObserver(
-            router=self, event_name=DIALOG_EVENT_NAME
-        )
+    pass
 
 
 class DialogRegistry(DialogRegistryProto):
     def __init__(self, dp: Dispatcher, dialogs: Sequence[ManagedDialogProto] = ()):
         super().__init__()
         self.dp = dp
```

Comment:
maybe add checking `dp` type, if not `Dispatcher` raise Error

Reply:
I'd rather replace it with router